pkg/sdk/testint/secrets_gen_integration_test.go

```diff
@@ -121,7 +120,7 @@ func TestInt_Secrets(t *testing.T) {
 		})
 	})
 
-	// regarding the https://docs.snowflake.com/en/sql-reference/sql/create-secret secret with empty oauth_scopes list should inherit scopes from security_integration, but it does not
+	// TODO [SNOW-1678756]: oauth_copes not inherited or not displayed correctly when not provided
```

Comment:
Nit: oauth_scopes

Reply:
Will correct it with next Secret related PR